bitrise.yml

```diff
@@ -28,6 +33,7 @@ workflows:
             - content: |-
                 #!/bin/bash
                 set -ex
+                touch app.apk
```

Comment:
you need a `app.apk` in your repository and to use it here: https://github.com/DataDog/dd-sdk-android/pull/83/files#diff-801b81b028645c52cf0443365fea2f93R41

Reply:
I think I will not do this for now. I don't want to keep a dummy apk there only for Bitrise :D